### ISSUE TITLE
Refresh issue in watchkit

### ISSUE THREAD

**Issue by moh1122** [jdrambarean/HelvetiSteps#3]
Hello
There is a refresh issue also on the watch kit?

**Comment by jdrambarean** [comment]
Yeah there is something relating to scheduling processor time that I need to look into. 

Best,

Joseph Drambarean
